docs/modules/ROOT/pages/architecture.adoc

```diff
@@ -4,17 +4,17 @@ ifdef::env-github[]
 :imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
 endif::[]
 
-A Grafana-based stack has been selected for a few reasons:
+Couchbase monitoring and observability stack [CMOS] is built primarily with Grafana-based components mainly for the following reasons:
 
 * Already in use with Prometheus exporter and similar on Couchbase Server
 * Industry standard and OSS with various integration points for other pipelines
 * Existing options to scale from single node up to cloud scale easily
 
-We expose Prometheus endpoints already for node-level information on a Couchbase Server instance.
-Recently we have exposed logs using Fluent Bit, primarily for kubernetes based solutions but this can be deployed on-premise as well.
+We expose Prometheus endpoints already for node-level information on a Couchbase Server instance starting from Couchbase Server 7.0 release.
+Recently we have exposed logs using Fluent Bit, primarily for Kubernetes based solutions but this can be deployed on-premise deployment as well.
```

Comment:
Find this a bit confusing with the addition of `deployment` at the end, though I may not have enough understanding of the CMOS setup. Previous wording seemed fine, aside from Kubernetes not being capitalized ofcourse.

Reply:
I agree @maria-robobug I'm updating the statements.